src/main/java/io/cdap/plugin/snowflake/common/util/SchemaHelper.java

```diff
@@ -95,6 +121,26 @@ private static Schema getParsedSchema(String schema) {
     }
   }
 
+  private static Schema getSchema(SnowflakeAccessor snowflakeAccessor, String tableName, String importQuery) {
```

Comment:
Add @Nullable in front of tableName and importQuery where ever it can be null in input params